src/components/data-details/index.js

```diff
@@ -1,48 +1,121 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import ItemAttribute from '../item-attribute'
+import FormAttribute from '../form-attribute'
 
 import './styles.scss'
 
-const DataDetails = ({
-  children,
-  formattedData,
-  type,
-}) => (
-  <div className="data-details">
-    {formattedData.map((row, idx) => (
-      <div className="row" key={idx}>
-        {row.map(item => (
-          <ItemAttribute
-            attribute={item.attribute}
-            key={item.attribute}
-            value={item.value}
-            type={type}
-          />
-        ))}
+export default class DataDetails extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      selectedEntity: null,
+    }
+  }
+
+  componentWillMount() {
+    this.setState({selectedEntity: this.props.currentEntity})
```

Comment:
A mettre dans le constructeur, sans le `setState`

Reply:
(this.props == props dans le constructeur)

Reply:
Done